lib/src/cli.rs

```diff
@@ -99,6 +100,11 @@ pub(crate) struct SwitchOpts {
 
     /// Target image to use for the next boot.
     pub(crate) target: String,
+
+    /// Make the switch into a custom stateroot. If the stateroot doesn't exist, it will be created
+    /// and `/var` of the current stateroot will be copied (copy-on-write) into it.
+    #[clap(long)]
+    pub(crate) stateroot: Option<String>,
```

Comment:
In the interest of forward progress, how about adding `#[clap(hide = true))]` and calling this `experimental_clone_into_stateroot` so it can be an experimental feature for now while we decide on how we stabilize this?